index.html

```diff
@@ -38,5 +38,118 @@ <h2 class="text-container" id="sdsection">
       <img src="arrow-button.svg" class="arrow-button"><a href="#"></a></img>
       </div>
     </section>
+    <section class="works">
+      <h1 class="recent-works">My Recent<br>Works</h1>
```

Comment:
- [ ] [OPTIONAL] It is a good html practice to stick to having only 1 `h1` in your document. This ensures that there is no conflicting or misleading info for search engines on what the subject matter of a webpage is all about. Could you use other heading tags here between `h2` - `h6`?